_source/organisms/dashboards/Dashboards.scss

```diff
@@ -118,6 +132,14 @@
         color: inherit;
         cursor: default;
       }
+
+      &.dashboards__item--dark-mode {
+        background-color: inherit;
+
+        .dashboards__label {
+          color: #fff;
+        }
+      }
```

Comment:
Just wondering, it's not possible to also use BEM here, right? Else you'd do that?

Reply:
For the variable?